src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -1,20 +1,78 @@
 package codeu.controller;
+
 import java.io.IOException;
+
 import javax.servlet.ServletException;
+
 import javax.servlet.http.HttpServlet;
+
 import javax.servlet.http.HttpServletRequest;
+
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
```

Comment:
Keep all the imports together to make it more compact, so remove the extra lines in between the.